.prettierrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file is using a big number of spaces. Please format it again with just `2` spaces indentation using `prettier`.

Reply:
Ok done